etc/containerpilot.json

```diff
@@ -21,8 +21,8 @@
   "coprocesses": [{{ if .CONSUL_AGENT }}
     {
       "command": ["/usr/local/bin/consul", "agent",
-                  "-data-dir=/data",
-                  "-config-dir=/config",
+                  "-data-dir=/var/lib/consul",
+                  "-config-dir=/etc/consul",
```

Comment:
Right. I'd normally want to conform to convention, but those original dir paths feel really unconventional in the context of the other paths we're we're using in this project.